lms/static/sass/views/_bookmarks.scss

```diff
@@ -0,0 +1,141 @@
+// Rules for placing bookmarks and search button side by side
+.wrapper-course-modes {
+  border-bottom: 1px solid lightgray;
+  display: table;
+  width: 98%;
+  padding: 6px 3px;
+
+  > div {
+    display: table-cell;
+  }
+}
+
+
+// Rules for Bookmarks Button
+.courseware-bookmarks-button {
+  width: 40%;
+  padding: 0 3px;
+  vertical-align: top;
+
+  button[aria-pressed="true"] {
+    &:before {
+      content: "\f02e";
+      font-family: FontAwesome;
+    }
+
+    background-color: $white;
+  }
+
+  button[aria-pressed="false"] {
+    &:before {
+      content: "\f097";
+      font-family: FontAwesome;
+    }
+
+    background: none;
+  }
+
+  .bookmarks-button {
+    background: none;
+    border-radius: ($baseline/4);
+    padding: 9px 8px 10px;
+    width: 100%;
+
+    &:hover {
+      background: none !important;
+    }
+
+    &:focus, &:active {
+      box-shadow: none !important;
+    }
+  }
+}
+
+
+// Rules for Bookmarks Results Header
+.bookmarks-results-header {
+  margin: 0;
+}
+
+// Rules for Bookmarks Results
+.bookmarks-results-list {
+  padding-top: $baseline;
+
+  .bookmarks-results-list-item {
+    border: 1px solid $gray;
+    padding: ($baseline/4) $baseline ($baseline/2);
+    border-radius: ($baseline/4);
+    margin-bottom: $baseline;
+    display: block;
+
+    &:hover {
+      border-color: $link-color;
+      color: $link-color;
+    }
+  }
+
+  .results-list-item-view {
+    @include float(right);
+    margin-top: $baseline;
+  }
+
+  .list-item-date {
+    color: $gray;
+    font-size: 14px;
+  }
+
+  a.bookmarks-results-list-item:before {
+    content: "\f02e";
+    position: relative;
+    top: -7px;
+    font-family: FontAwesome;
+  }
+
+  .list-item-content {
+    overflow: hidden;
+  }
+
+  .list-item-left-section {
+    @include float(left);
+
+    h3, p {
+      font-weight: 100;
+    }
+  }
+
+  .list-item-right-section {
+    @include float(right);
+    margin-top: 7px;
+  }
+}
+
+
+// Rules for empty bookmarks list
+.bookmarks-empty {
```

Comment:
NIT: from a naming standpoint, it might be helpful to make this set of styles nested within bookmarks results list, as shown below:

 .bookmarks-results-list  { &.is-empty { } }